test_aodndata/soop/test_soop_trv.py

```diff
@@ -79,7 +79,7 @@ def test_delete_from_wfs(self, mock_webfeatureservice):
 
         broker = WfsBroker(self.config.pipeline_config['global']['wfs_url'])
 
-        files_for_layer = broker.query_urls_for_layer('soop_trv_duplicate_url')
+        files_for_layer = broker.query_files('soop_trv_duplicate_url')
         self.assertEqual(files_for_layer[0],
                          'IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20181205T035932Z_END-20181206T045722Z/temperature/IMOS_SOOP-TRV_T_20181205T035932Z_VMQ9273_FV01_END-20181206T045722Z.nc')
```

Comment:
@lwgordonimos  Should this test be creating a `WfsBroker` instead of using the new `state_query` api?
e.g.
```suggestion
        handler = self.handler_class(GOOD_NC)
        files_for_layer = handler.state_query.query_wfs_files('soop_trv_duplicate_url')
        self.assertEqual(files_for_layer[0].dest_path,
                         'IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20181205T035932Z_END-20181206T045722Z/temperature/IMOS_SOOP-TRV_T_20181205T035932Z_VMQ9273_FV01_END-20181206T045722Z.nc')
```

In fact, this is already implicitly tested by `test_good_netcdf` above, so this one is kind of rendundant.
@lbesnard